helsingborg-stad/app-mitt-helsingborg
# Feat/optimize asset sizes

## Explain the changes you’ve made
Reduce sizes of our assets, saving 1334 KB

## Explain why these changes are made
Optimizing assets sizes decrease bundle size and (optionally) application performance

## Explain your solution
Run almost all images in a size optimizer platform and replaced images in the application.

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Run the application as usual, images should look like before

## Tested environments

- [] Storybook on a iOS device/simulator.
